Cargo.lock

```diff
@@ -806,16 +790,16 @@ dependencies = [
 
 [[package]]
 name = "h2"
-version = "0.3.24"
+version = "0.4.3"
```

Comment:
@lmammino and @toadslop , it seems that something isn't quite right with the "Audit" job used in this repo.

Note that this revision says "0.4.3", whereas the recommend solution is a later version.
```
Solution:  Upgrade to ^0.3.26 OR >=0.4.4
```
I discovered this because we build oidc-authorizer ourselves and run cargo audit directly, instead of using the GH action.  Independent verification FTW?

Reply:
Also, I guess I gotta scrutinize PRs a bit more that resolve issues via "cargo update"?